simulation/sub8_gazebo/worlds/contact.world

```diff
@@ -84,14 +91,18 @@
           </geometry>
         </collision>
       </link>
-    </model>
+    </model> -->
+    <include>
```

Comment:
What happens? Jerky control, like we saw when we tried to do 5x speed simulation?

If so, try catkin_make-ing this latest pr and see if it helps